examples/hooks-e-commerce/App.tsx

```diff
@@ -0,0 +1,369 @@
+
+            <div className="container-body">
+              <Panel header="Category">
+                <HierarchicalMenu
+                  attributes={[
+                    'hierarchicalCategories.lvl0',
+                    'hierarchicalCategories.lvl1',
+                  ]}
+                />
+              </Panel>
+
+              <Panel header="Brands">
+                <RefinementList
+                  attribute="brand"
+                  searchable={true}
+                  searchablePlaceholder="Search for brands…"
+                />
+              </Panel>
+
+              <Panel header="Price">
+                <PriceSlider attribute="price" />
+              </Panel>
+
+              <Panel header="Free shipping">
+                <ToggleRefinement
+                  attribute="free_shipping"
+                  label="Display only items with free shipping"
+                  on={true}
+                />
+              </Panel>
+
+              <Panel header="Ratings">
+                <Ratings attribute="rating" />
+              </Panel>
+            </div>
+          </section>
+
+          <footer className="container-filters-footer" data-layout="mobile">
+            <div className="container-filters-footer-button-wrapper">
+              <ClearFiltersMobile containerRef={containerRef} />
+            </div>
+
+            <div className="container-filters-footer-button-wrapper">
+              <SaveFiltersMobile onClick={closeFilters} />
+            </div>
+          </footer>
+        </div>
+
+        <section className="container-results">
+          <header className="container-header container-options">
+            <SortBy
+              className="container-option"
+              items={[
+                {
+                  label: 'Sort by featured',
+                  value: 'instant_search',
+                },
+                {
+                  label: 'Price ascending',
+                  value: 'instant_search_price_asc',
+                },
+                {
+                  label: 'Price descending',
+                  value: 'instant_search_price_desc',
+                },
+              ]}
+            />
+
+            <HitsPerPage
+              className="container-option"
+              items={[
+                {
+                  label: '16 hits per page',
+                  value: 16,
+                  default: true,
+                },
+                {
+                  label: '32 hits per page',
+                  value: 32,
+                },
+                {
+                  label: '64 hits per page',
+                  value: 64,
+                },
+              ]}
+            />
+          </header>
+
+          <Hits hitComponent={Hit} />
+          <NoResults />
+
+          <footer className="container-footer">
+            <Pagination
+              padding={2}
+              showFirst={false}
+              showLast={false}
+              // translations={{
```

Comment:
maybe this requires a custom widget, maybe we leave this part of the ui out until there's translations?